### ISSUE TITLE
Could not find file 'C:\Users\User\Opt\Simple-Bookmark\ssl\cert.pem'

### ISSUE THREAD

**Issue by scillidan** [VnUgE/Simple-Bookmark#1]
Dude, `task -- --setup` return errors on Windows 10:

![image](https://github.com/user-attachments/assets/8d92a1dc-7b7b-460b-99bd-cb4c01b1529e)

![image](https://github.com/user-attachments/assets/be33df92-f221-4c2e-832b-ae5122054a83)

**Comment by VnUgE** [comment]
hey @scillidan sorry for the delay, I've been fighting with my ISPs on mail servers so I missed the notification.

Yes, during the setup process on Windows no certificate is generated it must be done manually. I plan to update the task file in the future. For now, you should take a look at the Powershell Docs for [New-SelfSignedCertificate](https://learn.microsoft.com/en-us/powershell/module/pki/new-selfsignedcertificate) to get your new certificate created! 

I created an [internal issue](https://www.vaughnnugent.com/Resources/Software/Modules/Simple-Bookmark-issues?id=49) for this so I can hopefully add it to the next release. I'm currently fighting with storage so the website might be down when you look at it, but trust me it's there! Documentation will be updated as soon as I'm done fighting with the site!

**Comment by scillidan** [comment]
Thanks for your time :) I will try it again after future updates.
I have been using Linkding on Windows 10 since last year. I really like this tool. But deploying it is complex, especially on Windows. And I left a [note](https://scillidan.github.io/notes/serve/linkding.html) (I plan to re-write it in the future, make it more useful as a tiny manual) when I deploying it.
Because I feel `docker` on Windows is less stable, I didn't use docker container finally. I used `.cmd` command as boot script to run it liked a serve.
As you said in `README`, deploy methods other than `docker` can bring more nice and faster speeds :rocket:, it's a good way.

**Comment by VnUgE** [comment]
You are right and I appreciate you giving it a try! I care about bare-metal deployments, for speed as well as user freedom so you have the choice to just run on your desktop or server if you choose without needing to install a container runtime as you mentioned!
